### PR TITLE
show generated api key on list page, add copy to clipboard, reverse a…

### PR DESCRIPTION
**Pull request by VasoOppelt**
…pi keys order

### CODE REVIEW

File: src/app.sass [hunk 1]
```diff
@@ -23,8 +23,12 @@ html, body, .app
...
 	box-sizing: border-box
 
-.text-center
-	text-align: center
+.text
+	&-center
+		text-align: center
+
+	&-left
+		text-align: left !important
```
Comment (ViliamKopecky) [inline src/app.sass]: This can surely be solved without `!important`

File: src/components/ApiKeys.tsx [hunk 2]
```diff
@@ -71,8 +76,14 @@ export class Row extends React.Component<RowProps> {
 			typeof this.props.type !== 'undefined'
 				? apiKeyTypeClassnames[this.props.type]
 				: 'apiKey-type-round-label-none'
+		let apiKey = undefined
+		if (window.recentlyAddedApiKeys && this.props.id){
+			apiKey = window.recentlyAddedApiKeys[this.props.id]
+			window.recentlyAddedApiKeys.splice( window.recentlyAddedApiKeys.indexOf('this.props.id'), 1 );
```
Comment (ViliamKopecky) [inline src/components/ApiKeys.tsx]: Are you sure this works?

File: src/components/AddApiKey.tsx [hunk 3]
```diff
@@ -102,8 +64,12 @@ export default class AddApiKeys extends Component<AddApiKeysProps, AddApiKeysSta
 						name: '',
 						type: 0,
 						working: false,
-						recentlyAddedApiKeys: [...state.recentlyAddedApiKeys, newKey],
 					}))
+					if (!window.recentlyAddedApiKeys){
+						window.recentlyAddedApiKeys = []
+					}
+					window.recentlyAddedApiKeys[newKey.id] = newKey.token
```
Comment (ViliamKopecky) [inline src/components/AddApiKey.tsx]: It is not a good practice to use `Array` with id as index - You should use `object` for that.

**Comment by ViliamKopecky** [comment]
You dont need to convert SVG to react component anymore. There is a webpack loader for that.

For example:

```tsx
import { ReactComponent as GithubIcon } from '@fortawesome/fontawesome-free/svgs/brands/github.svg'
```

File: src/app.sass (hunk 1 above)
Comment (VasoOppelt) [reply src/app.sass]: done

File: src/components/ApiKeys.tsx (hunk 2 above)
Comment (VasoOppelt) [reply src/components/ApiKeys.tsx]: done

File: src/components/AddApiKey.tsx (hunk 3 above)
Comment (VasoOppelt) [reply src/components/AddApiKey.tsx]: done

**Pull request merged**